exercises/roman-numerals/introduction.md

```diff
@@ -0,0 +1,33 @@
+# Description
+
+Today, most of the world use Arabic numerals.
+But if you go back two thousand years, a large proportion of Europe were using Roman numerals instead.
```

Comment:
```suggestion
But if you go back two thousand years, a large proportion of Europe was using Roman numerals.
```

Reply:
I also feel like this is were as I'm talking about the people using the numerals, who are plural.

Reply:
Is it one large proportion, or several large proportion?